client/src/Components/About/About.js

```diff
@@ -114,13 +125,14 @@ const About = () => {
           formatSection(section, index % 2 !== 0)
         )}
         {!user && (
-          <Grid item className={style.center} xs={12}>
+          <Grid item className={style.footer} xs={12}>
             <Typography gutterBottom variant="body1">
               {t('about.signUp')}
             </Typography>
             <Button
               variant="contained"
               color="primary"
+              className={style.button}
```

Comment:
For consistency with the design of the whole app, this button shouldn't be further styled.